server/src/contact/contact.service.ts

```diff
@@ -35,9 +35,13 @@ export class ContactService {
 
       return firstRecord;
     } catch(e) {
-      const errorMessage = `Error finding contact by ID. ${e.message}`;
+      const errorMessage = `Error finding contact by ID, possibly due to missing or bad ID. ${e.message}`;
       console.log(errorMessage);
-      throw new HttpException(errorMessage, HttpStatus.UNAUTHORIZED);
+      throw new HttpException({
```

Comment:
Why is this formatted differently from the ones in `auth.service.ts`?

Reply:
Ah, good catch. I should format it the same as auth.service.ts!

Reply:
Fixed! Thanks!